_includes/head.html

```diff
@@ -19,30 +19,30 @@
     <title>{% if page.title %}{{ page.title | escape }}{% else %}{{ site.title | escape }}{% endif %}</title>
     <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
     <!-- Essential styles -->
-    <link rel="stylesheet" href="{{ site.path.plugins }}/bootstrap/css/bootstrap.min.css" type="text/css">
-    <link rel="stylesheet" href="{{ site.path.plugins }}/font-awesome/css/font-awesome.min.css" type="text/css">
-    <link rel="stylesheet" href="{{ site.path.plugins }}/fancybox/jquery.fancybox.css?v=2.1.5" media="screen">
+    <link rel="stylesheet" href="/assets/plugins/bootstrap/css/bootstrap.min.css" type="text/css">
+    <link rel="stylesheet" href="/assets/plugins/font-awesome/css/font-awesome.min.css" type="text/css">
+    <link rel="stylesheet" href="/assets/plugins/fancybox/jquery.fancybox.css?v=2.1.5" media="screen">
 
     <!-- Boomerang styles -->
-        <link id="wpStylesheet" type="text/css" href="{{ site.path.css }}/global-style.css" rel="stylesheet" media="screen">
+        <link id="wpStylesheet" type="text/css" href="/assets/css/global-style.css" rel="stylesheet" media="screen">
 
 
     <!-- Favicon -->
-    <link href="images/favicon.png" rel="icon" type="image/png">
+    <link href="/images/favicon.png" rel="icon" type="image/png">
 
     <!-- Assets -->
-    <link rel="stylesheet" href="{{ site.path.plugins }}/owl-carousel/owl.carousel.css">
-    <link rel="stylesheet" href="{{ site.path.plugins }}/owl-carousel/owl.theme.css">
-    <link rel="stylesheet" href="{{ site.path.plugins }}/sky-forms/css/sky-forms.css">
+    <link rel="stylesheet" href="/assets/plugins/owl-carousel/owl.carousel.css">
+    <link rel="stylesheet" href="/assets/plugins/owl-carousel/owl.theme.css">
+    <link rel="stylesheet" href="/assets/plugins/sky-forms/css/sky-forms.css">
     <!--[if lt IE 9]>
         <link rel="stylesheet" href="assets/sky-forms/css/sky-forms-ie8.css">
     <![endif]-->
 
     <!-- Required JS -->
-    <script src="{{ site.path.js }}/jquery.js"></script>
-    <script src="{{ site.path.js }}/jquery-ui.min.js"></script>
+    <script src="/assets/js/jquery.js"></script>
```

Comment:
this should be a variable path with a prepend url like {{ site.path.js | prepend: site.baseurl }}